src/fontcache.cpp

```diff
@@ -213,6 +225,8 @@ void InitFontCache(bool monospace)
 		LoadCoreTextFont(fs);
 #endif
 	}
+
+	DebugPrintFontSettings("End of initFontCache()");
```

Comment:
Seems unrelated to the actual task, and not really necessary.

There are coding-style issues in your Debug function too, space comes before `*` or `&`.

Reply:
I fixed the style issues.

I used this function a lot during the development of the original font chooser. I'm still using it while I work out the fontconfig code.  I imagine anyone else working on this code might also find it useful.